src/repositories/sql/create.sql

```diff
@@ -194,4 +194,18 @@ CREATE OR REPLACE RULE count_comments AS
         SELECT COUNT(*)
         FILTER (WHERE posts.post_id = comments.post_id AND comments.is_disabled = FALSE AND comments.is_deleted = FALSE)
         FROM comments)
-		WHERE new.post_id = posts.post_id;
\ No newline at end of file
+		WHERE new.post_id = posts.post_id;
+
+-- Create View
+
+CREATE VIEW general_overview AS
+    SELECT 
+        (SELECT COUNT (DISTINCT target_user_id) FROM reports) AS reported_users,
+        (SELECT COUNT (DISTINCT target_post_id) FROM reports) AS reported_posts,
+        (SELECT COUNT (DISTINCT target_comment_id) FROM reports) AS reported_comments,
+        (SELECT COUNT (target_id) FROM bans WHERE bans.expires_at > now()) AS banned_users,
+        (SELECT COUNT (post_id) FROM posts WHERE posts.status_id = 'ps1') AS pending_posts,
+        (SELECT COUNT (post_id) FROM posts WHERE posts.status_id = 'ps4') AS disabled_posts,
+        (SELECT COUNT (comment_id) FROM comments WHERE is_disabled = TRUE) AS disabled_comments,
+        (SELECT now()) AS issued_at;
+SELECT * FROM general_overview;
```

Comment:
I don't think this belongs here.

Reply:
Yes, thank you for reminding me. I accidentally added it there so the output will be seen.